Lib/importlib/_bootstrap_external.py

```diff
@@ -372,7 +372,7 @@ def _write_atomic(path, data, mode=0o666):
 # Whenever MAGIC_NUMBER is changed, the ranges in the magic_values array
 # in PC/launcher.c must also be updated.
 
-MAGIC_NUMBER = (3460).to_bytes(2, 'little') + b'\r\n'
+MAGIC_NUMBER = (3459).to_bytes(2, 'little') + b'\r\n'
```

Comment:
On Windows, PGO build error can be fixed with changing `PCbuild/regen.targets` (#27146) .
If the MAGIC_NUMBER revert is necessary for other reasons, a separated PR  (which also removes a comment at line 365) might be better as bpo-44530.

Reply:
This should not be here, thanks.